g_src/win32_compat.cpp

```diff
@@ -1,4 +1,5 @@
 #include <string>
+#include <unistd.h>
```

Comment:
This causes a failed compile on windows, since there's no unistd.h there and we do in fact compile this file on windows--not a major problem though